src/component/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import calculate from './logic/calculate';
+/* eslint-disable */
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {total: 0, next: null, operation: null}
+  }
+
+  handleClick = (event) => {
+    const buttonName = event.target.innerText;
+    const result = calculate(this.state, buttonName);
+    this.setState(result)
+  }
+  render() {
+    const {total, next, operation } = this.state;
+    return (
+      <div className="container">
+        <div className="mainDiv">
+          <button className="inputiv" type="submit">{total}{operation}{next}</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">AC</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">+/-</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">%</button>
+          <button className="btn orange" onClick={this.handleClick} type="submit">/</button>

```

Comment:
- [ ] Nice working on your operators, kindly check I get the error in the screenshot below when trying to divide you can use the `÷` symbol instead 
![image](https://user-images.githubusercontent.com/48371156/200096053-4ab0d22e-98b6-49ec-91c1-eb304ef721d6.png)

Reply:
Okay, Thanks.